lib/js/atoms/Stepper.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<div class="stepper">
+		<div v-for="(step, index) in steps" :key="index" class="stepper__item">
+			<div
+				v-if="index !== 0"
+				class="stepper__separator"
+				:class="{ '-active': step.isActive }"
+			></div>
+			<div class="stepper__icon" :class="{ '-active': step.isActive }">
```

Comment:
Is it intended that multiple steps are active at the same time? 
Maybe we should consider having only one step active, all steps before that step should be also marked green. Such encapsulation avoids leaking isActive logic to the application

Reply:
I thought about it, however I choose this solution

motivation:
 - this is DS component so should be as reusable and logic agnostic as possible
 - on the application level we gonna to write wrapper for applying that logic (if stepper will be only application component this 2 components should be merged)